index.html

```diff
@@ -10,10 +10,20 @@
     <div class="page">
         <!--header-->
         <div class="header">
+            <div class="left">
```

Comment:
in this context, "left" was not clear. reader did not understand what is left. it should be the name of component, more meaningful please